### DISCUSSION TITLE
Change Email template for Alerts

### DISCUSSION THREAD

**Discussion by anilkumardhiman** [datalust/seq-tickets#1576] [Help]
I am trying to send out a custom HTML template automatically from the alerts when an exception is encountered. But the template does not have all the properties like "Application", "SourceContext" and a lot of proerties which we see when we maually send a email from Seq. 
How can i get these properties in alert?
![image](https://user-images.githubusercontent.com/15940560/170736491-bce5bf7d-230e-41ad-86a6-c33b632041fc.png) 

I want my alert notification email to look like below :

![image](https://user-images.githubusercontent.com/15940560/170737031-647a60eb-8272-4134-8d7e-9217ee8db01a.png)

**Comment by tsimbalar** [comment]
Seems like the default template lives here : 

https://github.com/datalust/seq-app-htmlemail/blob/dev/src/Seq.App.EmailPlus/Resources/DefaultBodyTemplate.html

can this give you inspiration ? :P

**Reply by anilkumardhiman** [reply]
I have gone through this template. This does not give the output i am looking for. It gives the below results :

![image](https://user-images.githubusercontent.com/15940560/170742500-dc506861-f0ae-4dc8-9e68-52030200f6f8.png)

I am looking for this :
![image](https://user-images.githubusercontent.com/15940560/170742554-de9f5d13-03a1-4c33-9acc-527f9699737b.png)

**Reply by nblumhardt** [reply]
Hi! The difference here is that sending an individual event, and sending an alert notification, are quite different in Seq today.

You can include more information about the original event using the "Include contributing events" option when configuring the notification for the alert. This will include the original error message, though not the individual event properties.

Alternatively, you can create another instance of the HTML Email app, and choose "Stream incoming events" to send all errors, bypassing the alerting infrastructure. This will produce messages like your example.

Let me know if you need any more info.

**Reply by anilkumardhiman** [reply]
Thanks for the Response. But now i am getting both the alerts. How can i restrict them to only one alert

![image](https://user-images.githubusercontent.com/15940560/171203325-d6108873-f234-4349-a0af-acdf42082766.png)

I want only this alert to be triggered
![image](https://user-images.githubusercontent.com/15940560/171203384-d776953f-1c3e-421b-acd4-d3b70f4df22c.png)

**Reply by anilkumardhiman** [reply]
Also, I have a property named "Application" in the properties. How can i send it as part of the Subject in Alert?

{{$Application}} in Subject Template is not working

**Reply by nblumhardt** [reply]
Hi Anil; to get only one email, you can remove the alert under Settings > Alerts.

`{{Application}}` should work in the subject line (no `$` in front of the property name).

Let me know if this helps!
Nick

**Reply by anilkumardhiman** [reply]
Thanks Nick,, it is working now

**Reply by nblumhardt** [reply]
That's great, thanks for the heads-up 👍
